ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -294,7 +294,7 @@ const getHeadmarkerId = (data: Data, matches: NetMatches['HeadMarker']) => {
 export interface Data extends RaidbossData {
   readonly triggerSetConfig: {
     engravement1DropTower: 'quadrant' | 'clockwise' | 'tower';
-    classicalConceptsPairOrder: 'xsct' | 'cxts' | 'ctsx' | 'ctxs';
+    classicalConceptsPairOrder: 'xsct' | 'cxts' | 'ctsx' | 'ctxs' | 'shapeAndDebuff';
```

Comment:
Sorry for one last comment, but can you make this the default? I think this is a better default than bpog.

Reply:
> Sorry for one last comment, but can you make this the default? I think this is a better default than bpog.

😭

Reply:
Sorry! I think it's better to not make assumptions about strats if we can. I'll make a release note.